monai/apps/detection/transforms/dictionary.py

```diff
@@ -665,7 +666,7 @@ def __call__(self, data: Mapping[Hashable, NdarrayOrTensor]) -> Dict[Hashable, N
         d = dict(data)
 
         for key in self.image_keys:
-            d[key] = self.flipper(d[key])
+            d[key] = self.flipper(d[key])  # type: ignore
```

Comment:
I see many newly added `# type: ignore` in this file, is it possible to resolve the typehint?

Thanks.